lab_1_classify_by_unigrams/main.py

```diff
@@ -72,18 +110,54 @@ def compare_profiles(
 ) -> float | None:
     """
     Compare profiles and calculate the distance using symbols.
-
     Args:
         unknown_profile (dict[str, str | dict[str, float]]): A dictionary of an unknown profile
         profile_to_compare (dict[str, str | dict[str, float]]): A dictionary of a profile
             to compare the unknown profile to
-
     Returns:
         float | None: The distance between the profiles
-
     In case of corrupt input arguments or lack of keys 'name' and
     'freq' in arguments, None is returned
     """
+    # bad input check:
+    if (isinstance(unknown_profile, dict) and isinstance(profile_to_compare, dict) and
+            len(profile_to_compare) == 2 and len(unknown_profile) == 2):
```

Comment:
return None here

Reply:
why should I? I worry it will give me an error 'too many return statements'. Moreover, I cannot see the point to add return None here... Please, explain

Reply:
btw, about 10 instead of 8: unfortunately, I am not that smart to improve my code that much within 1 day until the last review. Hope, I will do that later to boost up my final mark. Maybe I will get sufficient experience for that ᕙ(⇀‸↼‶)ᕗ

Reply:
I am sure that you will gain experience and will complete the following works at 10!